e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -415,16 +415,21 @@ describe("scenarios > dashboard > tabs", () => {
     cy.get("@firstTabQuery").should("have.been.calledOnce");
     cy.get("@secondTabQuery").should("not.have.been.called");
 
+    getDashboardCard().findByText("User ID");
```

Comment:
maybe add a comment explaining why this is done? Just like you've described in the PR?